pkg/grpc/packet/server.go

```diff
@@ -0,0 +1,77 @@
+package pktgrpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/pbpacket"
```

Comment:
here too

Reply:
done in https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/beef4dd954e19bba5aec090685e889b2ee2e233f